cmd/tmsExport.go

```diff
@@ -38,7 +38,7 @@ func runTmsExport(exportConfig tmsExportOptions, communicationInstance tms.Commu
 		return errUploadDescriptors
 	}
 
-	_, errExportFileToNode := communicationInstance.ExportFileToNode(config.NodeName, fileId, config.CustomDescription, config.NamedUser)
+	_, errExportFileToNode := communicationInstance.ExportFileToNode(fileInfo, config.NodeName, config.CustomDescription, config.NamedUser)
```

Comment:
Would it not have made sense to add the fileInfo as the second argument? Or does it make more sense to have the fileInfo first, and then the nodeName?

Reply:
Motivation for that was that the rest of the arguments are the string ones. So I decided to group them togetehr after the `fileInfo` argument.

Reply:
got it